<?xml version="1.0" encoding="UTF-8"?>
<robot generator="Robot 3.2.2 (Python 3.6.2 on win32)" generated="20201210 15:07:23.210" rpa="false">
<suite id="s1" name="Batch Profile" source="C:\Users\palak\Downloads\coe-application-security\TestCases\Agent\API\ACMs\Batch_Profile.robot">
<test id="s1-t1" name="TC_012 Cross Origin Resource Sharing on Batch Profile">
<kw name="Log To Console" library="BuiltIn">
<doc>Logs the given message to the console.</doc>
<arguments>
<arg>"Performing Cross Origin Resource Sharing on Batch Profile"</arg>
</arguments>
<status status="PASS" starttime="20201210 15:07:24.797" endtime="20201210 15:07:24.797"></status>
</kw>
<kw name="Cross Origin Resource Sharing" library="API">
<arguments>
<arg>${Excel_Location}</arg>
<arg>Agent</arg>
<arg>ACMS_Batch_Profiles_API</arg>
</arguments>
<kw name="Log To Console" library="BuiltIn">
<doc>Logs the given message to the console.</doc>
<arguments>
<arg>"Executing Cross Origin Resource Sharing"</arg>
</arguments>
<status status="PASS" starttime="20201210 15:07:24.797" endtime="20201210 15:07:24.812"></status>
</kw>
<kw name="Log To Console" library="BuiltIn">
<doc>Logs the given message to the console.</doc>
<arguments>
<arg>${Excel_Location}</arg>
</arguments>
<status status="PASS" starttime="20201210 15:07:24.812" endtime="20201210 15:07:24.812"></status>
</kw>
<kw name="CORS" library="Cross_Origin_Resource_Sharing">
<arguments>
<arg>${Excel_Location}</arg>
<arg>${Excel_Sheet_Name}</arg>
<arg>${Module_Name}</arg>
</arguments>
<assign>
<var>${result}</var>
</assign>
<msg timestamp="20201210 15:07:24.850" level="INFO">Opening Excel
Opened Excel
Title of Sheet = Agent
Row 1 = API_Name
Row 2 = GATEWAY_API_POST
Row 3 = GATEWAY_API_GET
Row 4 = GATEWAY_API_DELETE
Row 5 = Mapping_API
Row 6 = Packages_API
Row 7 = Asset_API
Row 8 = ACMS_Summary_API
Row 9 = ACMS_Batch_Profiles_API
Module Found : ACMS_Batch_Profiles_API in Row - 9 of Worksheet - Agent
ACMS_Batch_Profiles_API
ACMS_Batch_Profiles_API

POST

http

internal-qaplatformagentconfigurationserv-1810934581.us-east-1.elb.amazonaws.com

/agent-configuration/v1/batchProfiles

http://internal-qaplatformagentconfigurationserv-1810934581.us-east-1.elb.amazonaws.com/agent-configuration/v1/batchProfiles
Expecting ',' delimiter: line 1 column 207 (char 206)
Data from find_vulnerable_parameters 
{}
'Body'
'Header'
'Cookie'
http://internal-qaplatformagentconfigurationserv-1810934581.us-east-1.elb.amazonaws.com/agent-configuration/v1/batchProfiles
/agent-configuration/v1/batchProfiles
POST
http
http://internal-qaplatformagentconfigurationserv-1810934581.us-east-1.elb.amazonaws.com/agent-configuration/v1/batchProfiles
local variable 'Body' referenced before assignment
local variable 'Header' referenced before assignment
local variable 'Cookie' referenced before assignment
{'Origin': 'www.geeksforgeeks.org'}
Error in executing HOST Injection</msg>
<msg timestamp="20201210 15:07:24.850" level="INFO">File "c:\python36\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "c:\python36\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "C:\Python36\Scripts\robot.exe\__main__.py", line 7, in &lt;module&gt;
    sys.exit(run_cli())
  File "c:\python36\lib\site-packages\robot\run.py", line 492, in run_cli
    return RobotFramework().execute_cli(arguments, exit=exit)
  File "c:\python36\lib\site-packages\robot\utils\application.py", line 46, in execute_cli
    rc = self._execute(arguments, options)
  File "c:\python36\lib\site-packages\robot\utils\application.py", line 83, in _execute
    rc = self.main(arguments, **options)
  File "c:\python36\lib\site-packages\robot\run.py", line 443, in main
    result = suite.run(settings)
  File "c:\python36\lib\site-packages\robot\running\model.py", line 259, in run
    self.visit(runner)
  File "c:\python36\lib\site-packages\robot\model\testsuite.py", line 174, in visit
    visitor.visit_suite(self)
  File "c:\python36\lib\site-packages\robot\model\visitor.py", line 88, in visit_suite
    suite.tests.visit(self)
  File "c:\python36\lib\site-packages\robot\model\itemlist.py", line 77, in visit
    item.visit(visitor)
  File "c:\python36\lib\site-packages\robot\model\testcase.py", line 78, in visit
    visitor.visit_test(self)
  File "c:\python36\lib\site-packages\robot\running\runner.py", line 134, in visit_test
    test.template).run_steps(test.keywords.normal)
  File "c:\python36\lib\site-packages\robot\running\steprunner.py", line 40, in run_steps
    self.run_step(step)
  File "c:\python36\lib\site-packages\robot\running\steprunner.py", line 61, in run_step
    return runner.run(step, context)
  File "c:\python36\lib\site-packages\robot\running\userkeywordrunner.py", line 56, in run
    return_value = self._run(context, kw.args, result)
  File "c:\python36\lib\site-packages\robot\running\userkeywordrunner.py", line 82, in _run
    exception, return_ = self._execute(context)
  File "c:\python36\lib\site-packages\robot\running\userkeywordrunner.py", line 153, in _execute
    StepRunner(context).run_steps(handler.keywords)
  File "c:\python36\lib\site-packages\robot\running\steprunner.py", line 40, in run_steps
    self.run_step(step)
  File "c:\python36\lib\site-packages\robot\running\steprunner.py", line 61, in run_step
    return runner.run(step, context)
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 56, in run
    return_value = self._run(context, kw.args)
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 79, in _run
    return self._run_with_output_captured_and_signal_monitor(runner, context)
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 101, in _run_with_output_captured_and_signal_monitor
    return self._run_with_signal_monitoring(runner, context)
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 106, in _run_with_signal_monitoring
    return runner()
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 94, in &lt;lambda&gt;
    return lambda: handler(*positional, **named)
  File "C:\Users\palak\Downloads\coe-application-security\MethodFiles\Cross_Origin_Resource_Sharing.py", line 127, in CORS
    returnvalue = readexcel(Excel_Location, Excel_Sheet_Name, Module_Name)
  File "C:\Users\palak\Downloads\coe-application-security\MethodFiles\Cross_Origin_Resource_Sharing.py", line 93, in readexcel
    traceback.print_stack()
  File "c:\python36\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "c:\python36\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "C:\Python36\Scripts\robot.exe\__main__.py", line 7, in &lt;module&gt;
    sys.exit(run_cli())
  File "c:\python36\lib\site-packages\robot\run.py", line 492, in run_cli
    return RobotFramework().execute_cli(arguments, exit=exit)
  File "c:\python36\lib\site-packages\robot\utils\application.py", line 46, in execute_cli
    rc = self._execute(arguments, options)
  File "c:\python36\lib\site-packages\robot\utils\application.py", line 83, in _execute
    rc = self.main(arguments, **options)
  File "c:\python36\lib\site-packages\robot\run.py", line 443, in main
    result = suite.run(settings)
  File "c:\python36\lib\site-packages\robot\running\model.py", line 259, in run
    self.visit(runner)
  File "c:\python36\lib\site-packages\robot\model\testsuite.py", line 174, in visit
    visitor.visit_suite(self)
  File "c:\python36\lib\site-packages\robot\model\visitor.py", line 88, in visit_suite
    suite.tests.visit(self)
  File "c:\python36\lib\site-packages\robot\model\itemlist.py", line 77, in visit
    item.visit(visitor)
  File "c:\python36\lib\site-packages\robot\model\testcase.py", line 78, in visit
    visitor.visit_test(self)
  File "c:\python36\lib\site-packages\robot\running\runner.py", line 134, in visit_test
    test.template).run_steps(test.keywords.normal)
  File "c:\python36\lib\site-packages\robot\running\steprunner.py", line 40, in run_steps
    self.run_step(step)
  File "c:\python36\lib\site-packages\robot\running\steprunner.py", line 61, in run_step
    return runner.run(step, context)
  File "c:\python36\lib\site-packages\robot\running\userkeywordrunner.py", line 56, in run
    return_value = self._run(context, kw.args, result)
  File "c:\python36\lib\site-packages\robot\running\userkeywordrunner.py", line 82, in _run
    exception, return_ = self._execute(context)
  File "c:\python36\lib\site-packages\robot\running\userkeywordrunner.py", line 153, in _execute
    StepRunner(context).run_steps(handler.keywords)
  File "c:\python36\lib\site-packages\robot\running\steprunner.py", line 40, in run_steps
    self.run_step(step)
  File "c:\python36\lib\site-packages\robot\running\steprunner.py", line 61, in run_step
    return runner.run(step, context)
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 56, in run
    return_value = self._run(context, kw.args)
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 79, in _run
    return self._run_with_output_captured_and_signal_monitor(runner, context)
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 101, in _run_with_output_captured_and_signal_monitor
    return self._run_with_signal_monitoring(runner, context)
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 106, in _run_with_signal_monitoring
    return runner()
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 94, in &lt;lambda&gt;
    return lambda: handler(*positional, **named)
  File "C:\Users\palak\Downloads\coe-application-security\MethodFiles\Cross_Origin_Resource_Sharing.py", line 174, in CORS
    traceback.print_stack()
  File "c:\python36\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "c:\python36\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "C:\Python36\Scripts\robot.exe\__main__.py", line 7, in &lt;module&gt;
    sys.exit(run_cli())
  File "c:\python36\lib\site-packages\robot\run.py", line 492, in run_cli
    return RobotFramework().execute_cli(arguments, exit=exit)
  File "c:\python36\lib\site-packages\robot\utils\application.py", line 46, in execute_cli
    rc = self._execute(arguments, options)
  File "c:\python36\lib\site-packages\robot\utils\application.py", line 83, in _execute
    rc = self.main(arguments, **options)
  File "c:\python36\lib\site-packages\robot\run.py", line 443, in main
    result = suite.run(settings)
  File "c:\python36\lib\site-packages\robot\running\model.py", line 259, in run
    self.visit(runner)
  File "c:\python36\lib\site-packages\robot\model\testsuite.py", line 174, in visit
    visitor.visit_suite(self)
  File "c:\python36\lib\site-packages\robot\model\visitor.py", line 88, in visit_suite
    suite.tests.visit(self)
  File "c:\python36\lib\site-packages\robot\model\itemlist.py", line 77, in visit
    item.visit(visitor)
  File "c:\python36\lib\site-packages\robot\model\testcase.py", line 78, in visit
    visitor.visit_test(self)
  File "c:\python36\lib\site-packages\robot\running\runner.py", line 134, in visit_test
    test.template).run_steps(test.keywords.normal)
  File "c:\python36\lib\site-packages\robot\running\steprunner.py", line 40, in run_steps
    self.run_step(step)
  File "c:\python36\lib\site-packages\robot\running\steprunner.py", line 61, in run_step
    return runner.run(step, context)
  File "c:\python36\lib\site-packages\robot\running\userkeywordrunner.py", line 56, in run
    return_value = self._run(context, kw.args, result)
  File "c:\python36\lib\site-packages\robot\running\userkeywordrunner.py", line 82, in _run
    exception, return_ = self._execute(context)
  File "c:\python36\lib\site-packages\robot\running\userkeywordrunner.py", line 153, in _execute
    StepRunner(context).run_steps(handler.keywords)
  File "c:\python36\lib\site-packages\robot\running\steprunner.py", line 40, in run_steps
    self.run_step(step)
  File "c:\python36\lib\site-packages\robot\running\steprunner.py", line 61, in run_step
    return runner.run(step, context)
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 56, in run
    return_value = self._run(context, kw.args)
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 79, in _run
    return self._run_with_output_captured_and_signal_monitor(runner, context)
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 101, in _run_with_output_captured_and_signal_monitor
    return self._run_with_signal_monitoring(runner, context)
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 106, in _run_with_signal_monitoring
    return runner()
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 94, in &lt;lambda&gt;
    return lambda: handler(*positional, **named)
  File "C:\Users\palak\Downloads\coe-application-security\MethodFiles\Cross_Origin_Resource_Sharing.py", line 180, in CORS
    traceback.print_stack()
  File "c:\python36\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "c:\python36\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "C:\Python36\Scripts\robot.exe\__main__.py", line 7, in &lt;module&gt;
    sys.exit(run_cli())
  File "c:\python36\lib\site-packages\robot\run.py", line 492, in run_cli
    return RobotFramework().execute_cli(arguments, exit=exit)
  File "c:\python36\lib\site-packages\robot\utils\application.py", line 46, in execute_cli
    rc = self._execute(arguments, options)
  File "c:\python36\lib\site-packages\robot\utils\application.py", line 83, in _execute
    rc = self.main(arguments, **options)
  File "c:\python36\lib\site-packages\robot\run.py", line 443, in main
    result = suite.run(settings)
  File "c:\python36\lib\site-packages\robot\running\model.py", line 259, in run
    self.visit(runner)
  File "c:\python36\lib\site-packages\robot\model\testsuite.py", line 174, in visit
    visitor.visit_suite(self)
  File "c:\python36\lib\site-packages\robot\model\visitor.py", line 88, in visit_suite
    suite.tests.visit(self)
  File "c:\python36\lib\site-packages\robot\model\itemlist.py", line 77, in visit
    item.visit(visitor)
  File "c:\python36\lib\site-packages\robot\model\testcase.py", line 78, in visit
    visitor.visit_test(self)
  File "c:\python36\lib\site-packages\robot\running\runner.py", line 134, in visit_test
    test.template).run_steps(test.keywords.normal)
  File "c:\python36\lib\site-packages\robot\running\steprunner.py", line 40, in run_steps
    self.run_step(step)
  File "c:\python36\lib\site-packages\robot\running\steprunner.py", line 61, in run_step
    return runner.run(step, context)
  File "c:\python36\lib\site-packages\robot\running\userkeywordrunner.py", line 56, in run
    return_value = self._run(context, kw.args, result)
  File "c:\python36\lib\site-packages\robot\running\userkeywordrunner.py", line 82, in _run
    exception, return_ = self._execute(context)
  File "c:\python36\lib\site-packages\robot\running\userkeywordrunner.py", line 153, in _execute
    StepRunner(context).run_steps(handler.keywords)
  File "c:\python36\lib\site-packages\robot\running\steprunner.py", line 40, in run_steps
    self.run_step(step)
  File "c:\python36\lib\site-packages\robot\running\steprunner.py", line 61, in run_step
    return runner.run(step, context)
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 56, in run
    return_value = self._run(context, kw.args)
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 79, in _run
    return self._run_with_output_captured_and_signal_monitor(runner, context)
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 101, in _run_with_output_captured_and_signal_monitor
    return self._run_with_signal_monitoring(runner, context)
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 106, in _run_with_signal_monitoring
    return runner()
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 94, in &lt;lambda&gt;
    return lambda: handler(*positional, **named)
  File "C:\Users\palak\Downloads\coe-application-security\MethodFiles\Cross_Origin_Resource_Sharing.py", line 186, in CORS
    traceback.print_stack()
  File "c:\python36\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "c:\python36\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "C:\Python36\Scripts\robot.exe\__main__.py", line 7, in &lt;module&gt;
    sys.exit(run_cli())
  File "c:\python36\lib\site-packages\robot\run.py", line 492, in run_cli
    return RobotFramework().execute_cli(arguments, exit=exit)
  File "c:\python36\lib\site-packages\robot\utils\application.py", line 46, in execute_cli
    rc = self._execute(arguments, options)
  File "c:\python36\lib\site-packages\robot\utils\application.py", line 83, in _execute
    rc = self.main(arguments, **options)
  File "c:\python36\lib\site-packages\robot\run.py", line 443, in main
    result = suite.run(settings)
  File "c:\python36\lib\site-packages\robot\running\model.py", line 259, in run
    self.visit(runner)
  File "c:\python36\lib\site-packages\robot\model\testsuite.py", line 174, in visit
    visitor.visit_suite(self)
  File "c:\python36\lib\site-packages\robot\model\visitor.py", line 88, in visit_suite
    suite.tests.visit(self)
  File "c:\python36\lib\site-packages\robot\model\itemlist.py", line 77, in visit
    item.visit(visitor)
  File "c:\python36\lib\site-packages\robot\model\testcase.py", line 78, in visit
    visitor.visit_test(self)
  File "c:\python36\lib\site-packages\robot\running\runner.py", line 134, in visit_test
    test.template).run_steps(test.keywords.normal)
  File "c:\python36\lib\site-packages\robot\running\steprunner.py", line 40, in run_steps
    self.run_step(step)
  File "c:\python36\lib\site-packages\robot\running\steprunner.py", line 61, in run_step
    return runner.run(step, context)
  File "c:\python36\lib\site-packages\robot\running\userkeywordrunner.py", line 56, in run
    return_value = self._run(context, kw.args, result)
  File "c:\python36\lib\site-packages\robot\running\userkeywordrunner.py", line 82, in _run
    exception, return_ = self._execute(context)
  File "c:\python36\lib\site-packages\robot\running\userkeywordrunner.py", line 153, in _execute
    StepRunner(context).run_steps(handler.keywords)
  File "c:\python36\lib\site-packages\robot\running\steprunner.py", line 40, in run_steps
    self.run_step(step)
  File "c:\python36\lib\site-packages\robot\running\steprunner.py", line 61, in run_step
    return runner.run(step, context)
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 56, in run
    return_value = self._run(context, kw.args)
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 79, in _run
    return self._run_with_output_captured_and_signal_monitor(runner, context)
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 101, in _run_with_output_captured_and_signal_monitor
    return self._run_with_signal_monitoring(runner, context)
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 106, in _run_with_signal_monitoring
    return runner()
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 94, in &lt;lambda&gt;
    return lambda: handler(*positional, **named)
  File "C:\Users\palak\Downloads\coe-application-security\MethodFiles\Cross_Origin_Resource_Sharing.py", line 259, in CORS
    traceback.print_stack()
  File "c:\python36\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "c:\python36\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "C:\Python36\Scripts\robot.exe\__main__.py", line 7, in &lt;module&gt;
    sys.exit(run_cli())
  File "c:\python36\lib\site-packages\robot\run.py", line 492, in run_cli
    return RobotFramework().execute_cli(arguments, exit=exit)
  File "c:\python36\lib\site-packages\robot\utils\application.py", line 46, in execute_cli
    rc = self._execute(arguments, options)
  File "c:\python36\lib\site-packages\robot\utils\application.py", line 83, in _execute
    rc = self.main(arguments, **options)
  File "c:\python36\lib\site-packages\robot\run.py", line 443, in main
    result = suite.run(settings)
  File "c:\python36\lib\site-packages\robot\running\model.py", line 259, in run
    self.visit(runner)
  File "c:\python36\lib\site-packages\robot\model\testsuite.py", line 174, in visit
    visitor.visit_suite(self)
  File "c:\python36\lib\site-packages\robot\model\visitor.py", line 88, in visit_suite
    suite.tests.visit(self)
  File "c:\python36\lib\site-packages\robot\model\itemlist.py", line 77, in visit
    item.visit(visitor)
  File "c:\python36\lib\site-packages\robot\model\testcase.py", line 78, in visit
    visitor.visit_test(self)
  File "c:\python36\lib\site-packages\robot\running\runner.py", line 134, in visit_test
    test.template).run_steps(test.keywords.normal)
  File "c:\python36\lib\site-packages\robot\running\steprunner.py", line 40, in run_steps
    self.run_step(step)
  File "c:\python36\lib\site-packages\robot\running\steprunner.py", line 61, in run_step
    return runner.run(step, context)
  File "c:\python36\lib\site-packages\robot\running\userkeywordrunner.py", line 56, in run
    return_value = self._run(context, kw.args, result)
  File "c:\python36\lib\site-packages\robot\running\userkeywordrunner.py", line 82, in _run
    exception, return_ = self._execute(context)
  File "c:\python36\lib\site-packages\robot\running\userkeywordrunner.py", line 153, in _execute
    StepRunner(context).run_steps(handler.keywords)
  File "c:\python36\lib\site-packages\robot\running\steprunner.py", line 40, in run_steps
    self.run_step(step)
  File "c:\python36\lib\site-packages\robot\running\steprunner.py", line 61, in run_step
    return runner.run(step, context)
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 56, in run
    return_value = self._run(context, kw.args)
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 79, in _run
    return self._run_with_output_captured_and_signal_monitor(runner, context)
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 101, in _run_with_output_captured_and_signal_monitor
    return self._run_with_signal_monitoring(runner, context)
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 106, in _run_with_signal_monitoring
    return runner()
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 94, in &lt;lambda&gt;
    return lambda: handler(*positional, **named)
  File "C:\Users\palak\Downloads\coe-application-security\MethodFiles\Cross_Origin_Resource_Sharing.py", line 273, in CORS
    traceback.print_stack()
  File "c:\python36\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "c:\python36\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "C:\Python36\Scripts\robot.exe\__main__.py", line 7, in &lt;module&gt;
    sys.exit(run_cli())
  File "c:\python36\lib\site-packages\robot\run.py", line 492, in run_cli
    return RobotFramework().execute_cli(arguments, exit=exit)
  File "c:\python36\lib\site-packages\robot\utils\application.py", line 46, in execute_cli
    rc = self._execute(arguments, options)
  File "c:\python36\lib\site-packages\robot\utils\application.py", line 83, in _execute
    rc = self.main(arguments, **options)
  File "c:\python36\lib\site-packages\robot\run.py", line 443, in main
    result = suite.run(settings)
  File "c:\python36\lib\site-packages\robot\running\model.py", line 259, in run
    self.visit(runner)
  File "c:\python36\lib\site-packages\robot\model\testsuite.py", line 174, in visit
    visitor.visit_suite(self)
  File "c:\python36\lib\site-packages\robot\model\visitor.py", line 88, in visit_suite
    suite.tests.visit(self)
  File "c:\python36\lib\site-packages\robot\model\itemlist.py", line 77, in visit
    item.visit(visitor)
  File "c:\python36\lib\site-packages\robot\model\testcase.py", line 78, in visit
    visitor.visit_test(self)
  File "c:\python36\lib\site-packages\robot\running\runner.py", line 134, in visit_test
    test.template).run_steps(test.keywords.normal)
  File "c:\python36\lib\site-packages\robot\running\steprunner.py", line 40, in run_steps
    self.run_step(step)
  File "c:\python36\lib\site-packages\robot\running\steprunner.py", line 61, in run_step
    return runner.run(step, context)
  File "c:\python36\lib\site-packages\robot\running\userkeywordrunner.py", line 56, in run
    return_value = self._run(context, kw.args, result)
  File "c:\python36\lib\site-packages\robot\running\userkeywordrunner.py", line 82, in _run
    exception, return_ = self._execute(context)
  File "c:\python36\lib\site-packages\robot\running\userkeywordrunner.py", line 153, in _execute
    StepRunner(context).run_steps(handler.keywords)
  File "c:\python36\lib\site-packages\robot\running\steprunner.py", line 40, in run_steps
    self.run_step(step)
  File "c:\python36\lib\site-packages\robot\running\steprunner.py", line 61, in run_step
    return runner.run(step, context)
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 56, in run
    return_value = self._run(context, kw.args)
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 79, in _run
    return self._run_with_output_captured_and_signal_monitor(runner, context)
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 101, in _run_with_output_captured_and_signal_monitor
    return self._run_with_signal_monitoring(runner, context)
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 106, in _run_with_signal_monitoring
    return runner()
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 94, in &lt;lambda&gt;
    return lambda: handler(*positional, **named)
  File "C:\Users\palak\Downloads\coe-application-security\MethodFiles\Cross_Origin_Resource_Sharing.py", line 286, in CORS
    traceback.print_stack()
  File "c:\python36\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "c:\python36\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "C:\Python36\Scripts\robot.exe\__main__.py", line 7, in &lt;module&gt;
    sys.exit(run_cli())
  File "c:\python36\lib\site-packages\robot\run.py", line 492, in run_cli
    return RobotFramework().execute_cli(arguments, exit=exit)
  File "c:\python36\lib\site-packages\robot\utils\application.py", line 46, in execute_cli
    rc = self._execute(arguments, options)
  File "c:\python36\lib\site-packages\robot\utils\application.py", line 83, in _execute
    rc = self.main(arguments, **options)
  File "c:\python36\lib\site-packages\robot\run.py", line 443, in main
    result = suite.run(settings)
  File "c:\python36\lib\site-packages\robot\running\model.py", line 259, in run
    self.visit(runner)
  File "c:\python36\lib\site-packages\robot\model\testsuite.py", line 174, in visit
    visitor.visit_suite(self)
  File "c:\python36\lib\site-packages\robot\model\visitor.py", line 88, in visit_suite
    suite.tests.visit(self)
  File "c:\python36\lib\site-packages\robot\model\itemlist.py", line 77, in visit
    item.visit(visitor)
  File "c:\python36\lib\site-packages\robot\model\testcase.py", line 78, in visit
    visitor.visit_test(self)
  File "c:\python36\lib\site-packages\robot\running\runner.py", line 134, in visit_test
    test.template).run_steps(test.keywords.normal)
  File "c:\python36\lib\site-packages\robot\running\steprunner.py", line 40, in run_steps
    self.run_step(step)
  File "c:\python36\lib\site-packages\robot\running\steprunner.py", line 61, in run_step
    return runner.run(step, context)
  File "c:\python36\lib\site-packages\robot\running\userkeywordrunner.py", line 56, in run
    return_value = self._run(context, kw.args, result)
  File "c:\python36\lib\site-packages\robot\running\userkeywordrunner.py", line 82, in _run
    exception, return_ = self._execute(context)
  File "c:\python36\lib\site-packages\robot\running\userkeywordrunner.py", line 153, in _execute
    StepRunner(context).run_steps(handler.keywords)
  File "c:\python36\lib\site-packages\robot\running\steprunner.py", line 40, in run_steps
    self.run_step(step)
  File "c:\python36\lib\site-packages\robot\running\steprunner.py", line 61, in run_step
    return runner.run(step, context)
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 56, in run
    return_value = self._run(context, kw.args)
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 79, in _run
    return self._run_with_output_captured_and_signal_monitor(runner, context)
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 101, in _run_with_output_captured_and_signal_monitor
    return self._run_with_signal_monitoring(runner, context)
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 106, in _run_with_signal_monitoring
    return runner()
  File "c:\python36\lib\site-packages\robot\running\librarykeywordrunner.py", line 94, in &lt;lambda&gt;
    return lambda: handler(*positional, **named)
  File "C:\Users\palak\Downloads\coe-application-security\MethodFiles\Cross_Origin_Resource_Sharing.py", line 312, in CORS
    traceback.print_stack()</msg>
<msg timestamp="20201210 15:07:24.850" level="INFO">${result} = {}</msg>
<status status="PASS" starttime="20201210 15:07:24.812" endtime="20201210 15:07:24.850"></status>
</kw>
<kw name="Log To Console" library="BuiltIn">
<doc>Logs the given message to the console.</doc>
<arguments>
<arg>${result}</arg>
</arguments>
<status status="PASS" starttime="20201210 15:07:24.850" endtime="20201210 15:07:24.850"></status>
</kw>
<kw name="List Should Not Contain Value" library="Collections">
<doc>Fails if the ``value`` is found from ``list``.</doc>
<arguments>
<arg>${result}</arg>
<arg>200</arg>
</arguments>
<status status="PASS" starttime="20201210 15:07:24.850" endtime="20201210 15:07:24.850"></status>
</kw>
<kw name="List Should Not Contain Value" library="Collections">
<doc>Fails if the ``value`` is found from ``list``.</doc>
<arguments>
<arg>${result}</arg>
<arg>201</arg>
</arguments>
<status status="PASS" starttime="20201210 15:07:24.850" endtime="20201210 15:07:24.850"></status>
</kw>
<kw name="List Should Not Contain Value" library="Collections">
<doc>Fails if the ``value`` is found from ``list``.</doc>
<arguments>
<arg>${result}</arg>
<arg>202</arg>
</arguments>
<status status="PASS" starttime="20201210 15:07:24.850" endtime="20201210 15:07:24.850"></status>
</kw>
<kw name="List Should Not Contain Value" library="Collections">
<doc>Fails if the ``value`` is found from ``list``.</doc>
<arguments>
<arg>${result}</arg>
<arg>302</arg>
</arguments>
<status status="PASS" starttime="20201210 15:07:24.850" endtime="20201210 15:07:24.850"></status>
</kw>
<status status="PASS" starttime="20201210 15:07:24.797" endtime="20201210 15:07:24.850"></status>
</kw>
<status status="PASS" starttime="20201210 15:07:24.797" endtime="20201210 15:07:24.850" critical="yes"></status>
</test>
<status status="PASS" starttime="20201210 15:07:23.212" endtime="20201210 15:07:24.850"></status>
</suite>
<statistics>
<total>
<stat pass="1" fail="0">Critical Tests</stat>
<stat pass="1" fail="0">All Tests</stat>
</total>
<tag>
</tag>
<suite>
<stat pass="1" fail="0" id="s1" name="Batch Profile">Batch Profile</stat>
</suite>
</statistics>
<errors>
<msg timestamp="20201210 15:07:23.488" level="ERROR">Error in file 'C:\Users\palak\Downloads\coe-application-security\Resources\API.robot' on line 3: Importing test library 'JSpringBotGlobal' failed: ModuleNotFoundError: No module named 'JSpringBotGlobal'
Traceback (most recent call last):
  None
PYTHONPATH:
  C:\Python36\Scripts\robot.exe
  c:\python36\python36.zip
  c:\python36\DLLs
  c:\python36\lib
  c:\python36
  c:\python36\lib\site-packages</msg>
</errors>
</robot>
